CollAction/Services/Image/AmazonS3ImageService.cs

```diff
@@ -118,6 +126,19 @@ private byte[] ConvertImageToPng(Image<Rgba32> image)
             }
         }
 
+        private double GetScaleRatioForImage(Image<Rgba32> image)
+        {
+            if(image.Width > _imageResizeThreshold || image.Height > _imageResizeThreshold){
```

Comment:
Formatting, use [allmann braces](https://en.wikipedia.org/wiki/Indentation_style#Allman_style)

Reply:
And spaces between ( and the "if"

Reply:
sory, fixed in 7a1eade